compare_core_with_daps_outputs/get_samples.py

```diff
@@ -0,0 +1,81 @@
+# Script to generate subsets of daps and core outputs for comparison
+
+repo_path = '/home/ubuntu/sky_workdir/asf_exploration/'  # path to asf_exploration folder
+
+import sys
+sys.path.append(repo_path)
+
+import polars as pl
+import s3fs
+from datetime import datetime
+import string
+from argparse import ArgumentParser
+import pathlib
+from tqdm import tqdm
+
+from compare_core_with_daps_outputs import config, schemas
+
+
+def convert_float(s):
+    "Convert floats in strings to int"
+    if s[-2:] == ".0":
+        s = s[0:-2]
+    return s
+
+
+def create_argparser():
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "--file_path",
+        help="Path to processed EPC file to use. Can be local file path or S3 URI."
+        )
+    
+    parser.add_argument(
+        "--pipeline",
+        help="Specify pipeline used to create file specified in `file_path` arg. Either `core` for `asf_core_data` or `daps` for `asf_daps`."
+    )
+
+    return parser
+
+
+if __name__ == "__main__":
+    
+    pl.enable_string_cache()
+    parser = create_argparser()
+    args = parser.parse_args()
+
+    today = datetime.today().strftime("%Y%m%d")
+
+    # Import processed dataset
+    if pathlib.Path(args.file_path).suffix == ".csv":
+        df = pl.read_csv(args.file_path, columns=config["common_cols"], dtypes=schemas.epc_schema)
+    elif pathlib.Path(args.file_path).suffix == ".parquet":
+        df = pl.read_parquet(args.file_path, columns=config["common_cols"])
```

Comment:
I wonder if you could see what happens when you add `use_pyarrow=True` arg.

For `pl.read_csv`, you will likely need to increase `infer_schema_length` from 100 to 10000